src/util/cluster.js

```diff
@@ -2,6 +2,7 @@ const cluster = require('cluster');
 const gracefulShutdown = require('http-graceful-shutdown');
 const util = require('util');
 const logger = require('../logger');
+const { redisConnector } = require('./redisConnector');
```

Comment:
We are getting the redisConnector here but not using it. Can't we make it initialize from here and use the initialized instance from transformer?

Reply:
Hmm let me try some approaches.

Reply:
Made some changes which seems to be working fine please check